modules/optimizing-mtu-networking.adoc

```diff
@@ -7,6 +7,12 @@
 
 There are two important maximum transmission units (MTUs): the network interface controller (NIC) MTU and the cluster network MTU.
 
-The NIC MTU is configured at the time of {product-title} installation, and you can also change the cluster's MTU as a Day 2 operation. See "Changing cluster network MTU" for more information. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+The NIC MTU is configured at the time of {product-title} installation, and you can also change the MTU of a cluster as a postinstallation task. See "Changing cluster network MTU" for more information. 
```

Comment:
See OpenShift documentation guidelines for [Lead-in link sentence style for non-xref situations](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations)

> For more information, see "<section_name>".

```suggestion
The NIC MTU is configured at the time of {product-title} installation, and you can also change the MTU of a cluster as a postinstallation task. For more information, see "Changing cluster network MTU".
```